nuvla/job_engine/job/base.py

```diff
@@ -148,14 +152,34 @@ def _init_statsd(self):
             except Exception as ex:
                 logging.error(f'Failed to initialise StatsD client for {self.args.statsd}: {ex}')
 
+    @staticmethod
+    def _write_cookies(cookie_file, cookies):
+        decoded_cookies = base64.b64decode(cookies.encode('utf-8')).decode('utf-8')
+        cookie_dir = os.path.dirname(cookie_file)
+        if not os.path.exists(cookie_dir):
+            os.mkdir(cookie_dir, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
+
+        with open(cookie_file, "w") as f:
+            f.write(decoded_cookies)
+
     def _init_nuvla_api(self):
         # true unless header authentication is used
-        reauthenticate = self.args.api_authn_header is None
+        cookies = os.getenv("JOB_COOKIES", "")
+        persist_cookie = cookies != "" and cookies is not None
```

Comment:
could be simplified with
```suggestion
        persist_cookie = bool(cookies)
```